docs/index.mdx

```diff
@@ -78,6 +78,12 @@ Find all the guides and resources you need to develop with Clerk.
+
+  ---
+
+  - [React Router (Beta)](/docs/quickstarts/react-router)
```

Comment:
is it beta? if so, we must add this tag to the docs in the sidebar, and some of the docs themselves. see ios for an example in the sidenav,
and its [quickstart](https://clerk.com/docs/quickstarts/ios)
and [overview](https://clerk.com/docs/references/ios/overview) - there's also a callout we'll have to add to docs as well

Reply:
The sidebar already has the Beta tags